README.md

```diff
@@ -5,6 +5,13 @@ This is a collection of Dockerfiles maintained by the [Kiwi.com Platform Team](h
 The images are built automatically by Docker Hub for the `:latest` tag,
 with updates of the base image triggering rebuilds.
 
+## kiwicom/black
+
+- Base image: `python:3.6-alpine3.7`
+- Packages: [`black`](https://github.com/ambv/black/)
+
+Image used to format python code and to check if all the files are correctly formatted on CI.
```

Comment:
I would explain that _we_ are using it mostly in CI. However, it's not its only purpose.

I would also add an example usage (`docker run ...`)